client/fileupload_test.go

```diff
@@ -0,0 +1,379 @@
+// Copyright (c) 2022 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0
+//
+// SPDX-License-Identifier: EPL-2.0
+
+package client
+
+import (
+	"encoding/json"
+	"log"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/eclipse/ditto-clients-golang/protocol"
+	mqtt "github.com/eclipse/paho.mqtt.golang"
```

Comment:
mqtt "github.com/eclipse/paho.mqtt.golang" - what is the purpose of this import. "mqtt" is the defined package name but because it don't follow the Go lang convention, it's used as: MQTT "github.com/eclipse/paho.mqtt.golang"